test/integration/driver.go

```diff
@@ -125,7 +125,6 @@ func setAuth(params infra) infra {
 	out := params
 	out.Name = "(auth infra)"
 	out.Auth = proxyconfig.ProxyMeshConfig_MUTUAL_TLS
-	out.Egress = false
```

Comment:
We can now remove both `Ingress` and `Egress` flags since they're always true. I can do it in a follow-up PR.